_posts/2016-07-14-polyrhythms.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "It's Like Patting Your Head and Rubbing Your Tummy: Polyrhythms"
+author: "Hayley MacLeod"
+categories: music
+header: /assets/img/2016-07-15/hypnosis.jpg
+---
+
+So you want to get better at rhythms, eh? Gain a little independence between your hands? Polyrhythms are a great way to start. A polyrhythm is two different rhythmic patterns played simultaneously against one another.
```

Comment:
I'd personally go for `So you want to get better at rhythms and gain a little independence between your hands?`